drivers/SmartThings/zigbee-lock/src/samsungsds/init.lua

```diff
@@ -46,7 +46,7 @@ local function unlock_cmd_handler(driver, device, command)
           DoorLock.ID,
           SAMSUNG_SDS_MFR_SPECIFIC_UNLOCK_COMMAND,
           SAMSUNG_SDS_MFR_CODE,
-          "1235"))
+          "\x10\x04\x31\x32\x33\x35"))
```

Comment:
This change seems unrelated?  Was it due to a failing test?

Reply:
some of these explicit hex values weren't copied over correctly to the public repo